usaspending_api/awards/v2/data_layer/orm_mappers.py

```diff
@@ -0,0 +1,186 @@
+        ("cfda_title", "cfda_title"),
+        # "Recipient" fields below
+        ("awardee_or_recipient_legal", "_recipient_name"),
+        ("awardee_or_recipient_uniqu", "_recipient_unique_id"),
+        ("ultimate_parent_legal_enti", "_parent_recipient_name"),
+        ("ultimate_parent_unique_ide", "_parent_recipient_unique_id"),
+        ("legal_entity_country_code", "_rl_location_country_code"),
+        ("legal_entity_country_name", "_rl_country_name"),
+        ("legal_entity_state_code", "_rl_state_code"),
+        ("legal_entity_city_name", "_rl_city_name"),
+        ("legal_entity_county_name", "_rl_county_name"),
+        ("legal_entity_address_line1", "_rl_address_line1"),
+        ("legal_entity_address_line2", "_rl_address_line2"),
+        ("legal_entity_address_line3", "_rl_address_line3"),
+        ("legal_entity_congressional", "_rl_congressional_code"),
+        ("legal_entity_zip_last4", "_rl_zip4"),
+        ("legal_entity_zip5", "_rl_zip5"),
+        ("legal_entity_foreign_posta", "_rl_foreign_postal_code"),
+        ("legal_entity_foreign_provi", "_rl_foreign_province"),
+        # "Place of Performance" fields below
+        ("place_of_perform_country_c", "_pop_location_country_code"),
+        ("place_of_perform_country_n", "_pop_country_name"),
+        ("place_of_perform_county_na", "_pop_county_name"),
+        ("place_of_performance_city", "_pop_city_name"),
+        ("place_of_perfor_state_code", "_pop_state_code"),
+        ("place_of_performance_congr", "_pop_congressional_code"),
+        ("place_of_perform_zip_last4", "_pop_zip4"),
+        ("place_of_performance_zip5", "_pop_zip5"),
+        ("place_of_performance_forei", "_pop_foreign_province"),
+    ]
+)
+
+
+FPDS_CONTRACT_FIELDS = OrderedDict(
+    [
+        ("idv_type_description", "idv_type_description"),
+        ("type_of_idc_description", "type_of_idc_description"),
+        ("referenced_idv_agency_iden", "referenced_idv_agency_iden"),
+        ("multiple_or_single_aw_desc", "multiple_or_single_aw_desc"),
+        ("solicitation_identifier", "solicitation_identifier"),
+        ("solicitation_procedures", "solicitation_procedures"),
+        ("number_of_offers_received", "number_of_offers_received"),
+        ("extent_competed", "extent_competed"),
+        ("other_than_full_and_o_desc", "other_than_full_and_o_desc"),
+        ("type_set_aside_description", "type_set_aside_description"),
+        ("commercial_item_acquisitio", "commercial_item_acquisitio"),
+        ("commercial_item_test_desc", "commercial_item_test_desc"),
+        ("evaluated_preference_desc", "evaluated_preference_desc"),
+        ("fed_biz_opps_description", "fed_biz_opps_description"),
+        ("small_business_competitive", "small_business_competitive"),
+        ("fair_opportunity_limi_desc", "fair_opportunity_limi_desc"),
+        ("product_or_service_code", "product_or_service_code"),
+        ("product_or_service_co_desc", "product_or_service_co_desc"),
```

Comment:
I believe this is being referred to as `product_or_service_desc` in the contract.  Perhaps the contract needs to be updated?

Reply:
Or the field should be renamed to match the contract

Reply:
API contract in PR does use `product_or_service_co_desc`